components/organisms/ContributorCard/contributor-card.tsx

```diff
@@ -58,15 +58,15 @@ const ContributorCard = ({ className, contributor, topic, repositories, range }:
           </div>
         </div>
         <div className="h-32">
-          <CardLineChart lineChartOption={chart} />
+          <CardLineChart lineChartOption={chart} height="140px" />
```

Comment:
`140px` seems to be the sweet spot for height as this item is a canvas.

![CleanShot 2023-12-13 at 14 50 43](https://github.com/open-sauced/app/assets/833231/b8f0cfa3-0be0-45d2-9836-bae42044bb2a)